packages/core/src/TolgeeConfig.ts

```diff
@@ -17,10 +17,31 @@ export class TolgeeConfig {
   apiKey?: string;
   inputPrefix?: string = '%-%tolgee:';
   inputSuffix?: string = '%-%';
+  /**
+   * Overrides all language settings
+   */
   forceLanguage?: string;
+  /**
+   * Overrides language auto detection
+   */
   defaultLanguage?: string = 'en';
+  /**
+   * Language which can be used for language detection
```

Comment:
Language -> Languages. 

localStorage is our internal way, how we do store the currentLanguage. I wouldn't know what does it mean without any context.